plugins/builds/steps/active.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const getSchema = schema.models.build.getStep;
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/builds/{id}/steps',
+    config: {
+        description: 'Get a step for a build',
+        notes: 'Returns a step record',
+        tags: ['api', 'builds', 'steps'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build', 'pipeline', '!guest', 'temporal']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const buildFactory = request.server.app.buildFactory;
+            const buildIdCred = request.auth.credentials.username
+                && request.auth.credentials.username.toString();
+            const buildId = request.params.id && request.params.id.toString();
+            const status = request.query.status;
+
+            if (status !== 'active') {
```

Comment:
if `status` param is not set, I would expect this endpoint to return all steps for a build

Reply:
Yeah, will add additional logic to the api based on status and step code as a separate feature.